academic_helper/views/schedule.py

```diff
@@ -81,13 +105,13 @@ def post(self, request, *args, **kwargs):
         if "search_val" in request.POST:
             search_val = request.POST["search_val"]
             return self.on_search_course(search_val)
-        if "course_number" in request.POST:
+        elif "course_number" in request.POST:
```

Comment:
There is a return before the `if`, so no need for `elif`, though I agree it is more readable.

Reply:
Oh yhe, actually just leftovers(I considered adding it as a POST at one point).